src/emscripten-optimizer/parser.cpp

```diff
@@ -41,6 +41,7 @@ IString TOPLEVEL("toplevel"),
         CONTINUE("continue"),
         SWITCH("switch"),
         STRING("string"),
+        TRY("try"),
```

Comment:
Yeah, I emit normal JS outside the asm.js module. This JS can call into the module's exported functions for correctness testing. The generated tests are not part of this CL because they are generated by the script. Even then, the tests including try...catch are not generated by the script because they do not pass since asm.js does not trap on things like divide by zero. You can generate and see these tests manually by doing `bin/wasm2asm --allow-asserts --pedantic test/spec/i32.wast`.